### PR TITLE
Changing some names to make more obvious

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Some updates to test, to have better names

### CODE REVIEW

**Review by pierDipi — APPROVED** [review]
/lgtm

**Review by pierDipi — APPROVED** [review]
/lgtm

File: test/extensione2e/kafka/source_kafka_broker_ksvc_test.go [hunk 1]
```diff
@@ -57,18 +57,18 @@ kind: %q`, channelAPIVersion, kafkaChannelKind),
...
 		},
 		Spec: eventingv1.TriggerSpec{
-			Broker: kafkaBrokerName,
+			Broker: kafkaChannelBrokerName,
 			Subscriber: duckv1.Destination{
 				Ref: &duckv1.KReference{
 					APIVersion: ksvcAPIVersion,
 					Kind:       ksvcKind,
-					Name:       helloWorldService + "-broker",
+					Name:       helloWorldService + "-kafkaChannelBroker",
```
Comment (markusthoemmes) [inline test/extensione2e/kafka/source_kafka_broker_ksvc_test.go:65]: The uppercase letters are breaking in the test.

Comment (matzew) [reply test/extensione2e/kafka/source_kafka_broker_ksvc_test.go:65]: ah, one more forgotten - saw that already

**Pull request merged**